api/python/quilt3/VERSION

```diff
@@ -0,0 +1 @@
+3.1.5
```

Comment:
We should really get this from `setup.py`, in the same directory, so as not to dupilcate state.

Reply:
State is not duplicated. `setup.py` get the version from this file as does `__init__`. This allows users to call `quilt3.__version__`.